cli/js/lib.deno.ns.d.ts

```diff
@@ -1388,8 +1388,9 @@ declare namespace Deno {
    * Requires `allow-read` permission. */
   export function realpath(path: string): Promise<string>;
 
-  export interface DirEntry extends FileInfo {
+  export interface DirEntry {
     name: string;
+    info: FileInfo;
```

Comment:
If I remember correctly we want to make `stat` optional - so this field would be optional as well; is that correct @piscisaureus? If so we should do that before 1.0 as making it `info?` is breaking change